Pages/Languages.cs

```diff
@@ -0,0 +1,141 @@
+﻿using Hemz_QA_Mars.common;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Support.UI;
+using System.Reflection.Emit;
+
+namespace Hemz_QA_Mars.Pages
+{
+    public class Languages : Driver
+    {
+        //Tab Locator
+        By languageTab = By.XPath("//a[@data-tab='first']");
+
+        // Common Locators
+        By languageField = By.XPath("//input[@placeholder='Add Language']");
```

Comment:
this can change all these to return IWebElement for example 

IWebElement languageTabElement = driver.FindElement(By.XPath("//a[@data-tab='first']"));

you can see the following example as well https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Reply:
please fix this in all pages

Reply:
As suggested, I have updated to return IWebElement in all places (refer below)
https://github.com/Godzeye99/Mars_QA_repo/blob/onboarding_task/Pages/LanguagesPage.cs#L26